docs/setup.asciidoc

```diff
@@ -3,19 +3,36 @@
 The .NET Agent ships as a set of NuGet packages via https://nuget.org[nuget.org].
 You can add the Agent to your .NET application by referencing one of these packages.
 
+On .NET Core the agent also supports auto instrumentation without any code change and without any recompilation of your projects. See section <<zero-code-change-setup,  Zero code change setup on .NET Core>> for more details.
+
+[float]
+== Get started
+
+* <<setup-dotnet-net-core>>
+* <<setup-asp-net-core>>
+* <<setup-asp-net>>
+* <<setup-ef6>>
+* <<setup-sqlclient>>
+* <<setup-general>>
```

Comment:
Is this an order that makes sense in the .NET world? Would we be better of alphabetizing these?

Reply:
The order was intentional - it goes (based on my own judgement) from more popular to less popular. I think alphabetic ordering would be less natural.

Reply:
Makes sense